install/check-minimum-requirements.sh

```diff
@@ -8,7 +8,8 @@ MIN_CPU_HARD=2
 MIN_CPU_SOFT=4
 
 DOCKER_VERSION=$(docker version --format '{{.Server.Version}}')
-COMPOSE_VERSION=$($dc --version | sed 's/docker-compose version \(.\{1,\}\),.*/\1/')
+# Do NOT use $dc instead of `docker-compose` below as older versions don't support certain options and fail
+COMPOSE_VERSION=$(docker-compose --version | sed 's/docker-compose version \(.\{1,\}\),.*/\1/')
```

Comment:
Does the "`docker-compose` → `docker compose`" warning/notice leak through here? If so, do we want to suppress it, perhaps by redirecting stderr?

Reply:
Nope:

```shell
$ docker-compose --version
docker-compose version 1.29.2, build 5becea4c
```